### PR TITLE
Case summary UAT changes

### PR DESCRIPTION
**Pull request by snopoke**
Easiest to review per commit

http://manage.dimagi.com/default.asp?158760

screenshot from 2015-03-10 13 00 04

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/app_summary.ng.js [hunk 1]
```diff
@@ -213,10 +220,14 @@
...
                 showCalculations: '=',
-                showLabels: '='
+                showLabels: '=',
+                lang: '='
             },
             controller: function ($scope) {
                 $scope.getIcon = utils.getIcon;
+                $scope.getQuestionLabel = function(question) {
+                   return utils.translateName(question.translations, $scope.lang, question.label);
+                }
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/app_summary.ng.js]: Missing semicolon.

**Comment by benrudolph** [comment]
Looks good to me, nice commit by commit breakdown. easy to follow

**Comment by snopoke** [comment]
@benrudolph :white_check_mark:

**Comment by czue** [comment]
when do we release this guy?

**Comment by snopoke** [comment]
pinged Amelia

**Pull request merged**